src/lib/pubkey/pem/pem.cpp

```diff
@@ -123,6 +123,23 @@ secure_vector<uint8_t> decode(DataSource& source, std::string& label)
    return base64_decode(b64.data(), b64.size());
    }
 
+std::vector<secure_vector<uint8_t>> decode_all(DataSource& source)
```

Comment:
It's not obvious this function is generally useful, so for now instead of adding it as a new public API just create it in an anon namespace for the cert store implementation.